pkg/server/manager/pipeline.go

```diff
@@ -0,0 +1,33 @@
+/*
+Copyright 2017 caicloud authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package manager
+
+import (
+	"github.com/caicloud/cyclone/store"
+)
+
+type PipelineManager interface {
+	// CRUD
+}
+
+type pipelineManager struct {
+}
+
+func NewPipelineManager(dataStore *store.DataStore) (PipelineManager, error) {
```

Comment:
comments

Reply:
Just a framework for pipeline, will add comments with implementation of pipeline in next PR.